build.xml

```diff
@@ -23,15 +23,21 @@
   <property name="cobertura.dir" value="${basedir}/../cobertura/"/>
   <!-- Load build environment specific properties. -->
   <property file="build.properties"/>
-  <property name="compile.java.source" value="6"/>
```

Comment:
why not leave the representation in place?  That is why not leave the two variables and have them both be at 7?